pages/event/[id].tsx

```diff
@@ -0,0 +1,41 @@
+import { GetServerSideProps } from 'next'
+import Footer from '../../components/Footer'
+import Layout from '../../components/Layout'
+import Container from '../../components/Layout/Container'
+import Navbar from '../../components/Navbar'
+import { PostProps } from '../event'
+
+interface SingleEventProps {
+  post: PostProps
+}
+
+const SingleEventPage = ({ post }: SingleEventProps): JSX.Element => {
+  const { title, body } = post
+  return (
+    <Layout title={`SLIIT RSVP | ${title}`}>
+      <Navbar />
+      <section
+        style={{ backgroundImage: 'url(/patterns/single-event.svg)' }}
+        className="bg-no-repeat bg-right-top bg-cover py-10"
+      >
+        <Container>{title}</Container>
+      </section>
+      <Footer />
+    </Layout>
+  )
+}
+
+export default SingleEventPage
```

Comment:
File naming is wrong. 
cc @Shehanka

Reply:
It okay.. I just got this 
https://nextjs.org/docs/basic-features/pages#pages-with-dynamic-routes